log4j-api/src/main/java/org/apache/logging/log4j/util/PropertiesUtil.java

```diff
@@ -435,17 +452,18 @@ private Environment(final PropertySource propertySource) {
                 // Access to System Properties is restricted so just skip it.
             }
             sources.add(propertySource);
-			for (final ClassLoader classLoader : LoaderUtil.getClassLoaders()) {
-				try {
-					for (final PropertySource source : ServiceLoader.load(PropertySource.class, classLoader)) {
-						sources.add(source);
-					}
-				} catch (final Throwable ex) {
-					/* Don't log anything to the console. It may not be a problem that a PropertySource
-					 * isn't accessible.
-					 */
-				}
-			}
+            for (final ClassLoader classLoader : LoaderUtil.getClassLoaders()) {
```

Comment:
This was indented with tabs.

Reply:
Never tabs, only spaces in this project; -)

Reply:
Okay, thank you so much.

Reply:
@garydgregory, would you mind replacing tabs with spaces in `log4j-core`? IIRC you already did it for `log4j-1.2-api`.